tools/buildtools/ambertools/antechamber.xml

```diff
@@ -0,0 +1,150 @@
+<tool id="antechamber" name="AnteChamber" version="@VERSION@">
+  <description>Amber's molecular input file processor</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements"></expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+        antechamber -i $input1 -o $output1 -fi $input_format.selected_format -fo $selected_output_format -c $allparams.c -pf $extraparams.pf
+        #if $extraparams.usenc
+        -nc $allparams.nc
+        #end if
+        -at $allparams.at -j $allparams.j -rn '$allparams.resname' -m $allparams.m
+        ]]>
+  </command>
+  <inputs>
+    <conditional name="input_format">
+      <param name="selected_format" type="select" label="Select a file format:" help="In PDB or mol2 format">
+        <option value="pdb" selected="true">pdb</option>
+        <option value="mol2">mol2</option>
+      </param>
+      <when value="pdb">
+        <param type="data" name="input1" format="pdb"/>
```

Comment:
this conditional is not needed, just write <param type="data" name="input1" format="mol2,pdb"/>

Reply:
UI 'breaks' when I remove the conditional or the when. The input selector doesn't show. What am I missing?

Reply:
`<param type="data" name="input1" format="pdb,mol2"/>` does not work?

Reply:
OK got it now. I remove the everything and just keep that line. MAGIC!

Reply:
Adding `label="Molecule input"` or similar would be good as well.